package.json

```diff
@@ -51,7 +51,6 @@
   },
   "dependencies": {
     "assert-never": "^1.2.1",
-    "brotli": "^1.3.3",
```

Comment:
This library has been broken on versions of node > 16. There's currently no tests in this codebase for anything brotli related 🙃 

Replacing this third-party library with the brotli implementation provided by the NodeJS builtin `zlib` module.